src/test/test_nvmeof_mon_encoding.cc

```diff
@@ -0,0 +1,194 @@
+// -*- mode:C++; tab-width:8; c-basic-offset:2; indent-tabs-mode:t -*-
+// vim: ts=8 sw=2 smarttab
+/*
+ * Ceph - scalable distributed file system
+ *
+ * Copyright (C) 2024
```

Comment:
```suggestion
 * Copyright (C) 2024 IBM, Inc.
```
here and everywhere else. Red Hat does not own the copyright on these changes.